models/keyword.go

```diff
@@ -1,92 +1,37 @@
 package models
 
 import (
-	"encoding/csv"
-	"errors"
-	"io"
-	"mime/multipart"
-	"os"
-	"path/filepath"
-
-	"github.com/gin-gonic/gin"
 	"github.com/gutakk/go-google-scraper/db"
-	errorHandler "github.com/gutakk/go-google-scraper/helpers/error_handler"
 	"gorm.io/gorm"
 )
 
-const (
-	fileFormatError         = "file must be CSV format"
-	fileLengthError         = "CSV file must contain between 1 to 1000 keywords"
-	invalidDataError        = "invalid data"
-	somethingWentWrongError = "something went wrong, please try again"
-)
-
 type Keyword struct {
 	gorm.Model
 	Keyword string `gorm:"notNull;index"`
 	UserID  uint
 	User    User
 }
 
-func ReadFile(filename string) ([]string, error) {
-	csvfile, openErr := os.Open(filename)
-	if openErr != nil {
-		return nil, errors.New(somethingWentWrongError)
-	}
+func GetKeywords(condition map[string]interface{}) ([]Keyword, error) {
```

Comment:
Since there is a condition, shouldn't it be `GetKeywordsBy` to be consistent with `FindUserBy`?

Reply:
Good catch. Fixed in 03e19f9187f1c95d2a64d46b5b9c727cc46eda49